### PR TITLE
so/manifesto-page-version-2.4

### LINKED ISSUE #4: Create page: Manifesto

### PR DESCRIPTION
**Pull request by sssoja**
### What changes have you made?
- added header component 

### Which issue(s) does this PR fix?

Fixes #4 

### Screenshots (if there are design changes)

Screenshot 2020-08-03 at 19 12 32
Screenshot 2020-08-03 at 19 12 47

### How to test
`yarn start` and nav to Manifesto

### CODE REVIEW

File: src/components/Manifesto.tsx [hunk 1]
```diff
@@ -2,111 +2,85 @@ import React from "react";
...
-      px={[1, 4, 6]}
-      pt={5}
-      display="flex"
-      alignItems="center"
-      justifyContent="center"
-    >
-      <Container p={4} display="block">
-        <H1 color="athensGray" fontSize={[2, 4, 5]} pb={5}>
+    <Main p={6} alignItems="center" justifyContent="center">
+      <Header />
```
Comment (Jbarget) [inline src/components/Manifesto.tsx:57]: ive just commented on one of the issues for adding the Header to all the pages. Its best if its done in one place in the parent of all the routes

File: src/components/Manifesto.tsx [hunk 2]
```diff
@@ -2,111 +2,85 @@ import React from "react";
...
+  ${grid};
 `;
 
-const Paragraph = styled.p<ColorProps & TypographyProps & SpaceProps>`
+const Paragraph = styled.p<TypographyProps & SpaceProps>`
   text-transform: uppercase;
-  ${color};
   ${typography};
   ${space};
+  padding-bottom: 20px;
```
Comment (Jbarget) [inline src/components/Manifesto.tsx:47]: this can be a prop on the Component since you're already using `space`

Comment (sssoja) [reply src/components/Manifesto.tsx:47]: It applies to all <P> tags though so I'm trying to keep it as DRY as poss i.e. say it once in the Component and void `pb`ing everywhere

Comment (Jbarget) [reply src/components/Manifesto.tsx:47]: sure, lets leave it like this but its good to be aware that if (in the unlikely case) the theme changed then this padding-bottom wouldnt change

Comment (sssoja) [reply src/components/Manifesto.tsx:47]: what's best practice? to use props in every case?

what if, for example, an element only uses one prop once like background color? (i know we're using Global Styles so probs bad example) do you pass the prop which is essentially more code or do you just go with old-fashioned CSS

File: src/components/Manifesto.tsx (hunk 1 above)
Comment (Jbarget) [reply src/components/Manifesto.tsx:56]: let's tackle this on a separate PR 👍

**Pull request merged**